retrieval/discovery/consul.go

```diff
@@ -276,6 +278,7 @@ func (cd *ConsulDiscovery) watchService(srv *consulService, ch chan<- *config.Ta
 				ConsulTagsLabel:           clientmodel.LabelValue(tags),
 				ConsulServiceAddressLabel: clientmodel.LabelValue(node.ServiceAddress),
 				ConsulServicePortLabel:    clientmodel.LabelValue(strconv.Itoa(node.ServicePort)),
+				ConsulServiceIDLabel:      clientmodel.LabelValue(node.ServiceID),
```

Comment:
I think ```ServiceID``` was the only item that was excluded.
See also:
https://github.com/hashicorp/consul/blob/c8f76b5e2cf6bfdc52af1924114e8dff40394821/website/source/docs/agent/http/catalog.html.markdown#-v1catalogserviceservice